npm/react/cypress/component/basic/rerender/effects.cy.jsx

```diff
@@ -35,7 +36,8 @@ it('should not run unmount effect cleanup when rerendering', () => {
   })
 })
 
-it('should run unmount effect cleanup when unmounting', () => {
+// TODO: is this still valid in React 18? Does re-rendering cause previous effects to be cleaned up?
+it.skip('should run unmount effect cleanup when unmounting', () => {
```

Comment:
I am not even sure if this test makes sense, I'm pretty sure re-rendering should not cause the return value from `useEffect` to be triggered - only when unmounting.